robotiq_cpp_control/include/gripper_interface.h

```diff
@@ -40,70 +40,106 @@
  *		1) C++ objects, as seen in rs_gripper_interface_test.cpp
  *		2) Publish commands on the ROS topic "grip_command".
  * @version 1.0
- * 
+ *
  * @author  Adam Allevato <adam.d.allevato@gmail.com>
  * @copyright BSD 3-paragraph
  * @date    Dec 4, 2015
  */
+namespace robotiq_cpp_control
+{
+
+namespace MODE
```

Comment:
Namespaces should be CamelCase.  Following the style guide, the namespace for this enum should be `Modes`.